config/settings/base.py

```diff
@@ -251,29 +253,6 @@
     "root": {"level": "INFO", "handlers": ["console"]},
 }
 
-# Celery
```

Comment:
Please restore the celery config for now.

Reply:
Done !